### PR TITLE
add phpstorm and php interpreter dockerfiles

### PR DESCRIPTION
**Pull request by sharkymark**
creating ubuntu and centos Dockerfiles for JetBrains PhpStorm IDE and these include a PHP interpreter.  Inherits multi-editor image for JetBrains browser IDE support.

### CODE REVIEW

File: images/phpstorm/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+[![Docker Pulls](https://img.shields.io/docker/pulls/codercom/enterprise-phpstorm?label=codercom%2Fenterprise-phpstorm)](https://hub.docker.com/r/codercom/enterprise-phpstorm)
+
+## Description
+
+Wraps [enterprise-multieditor](../multieditor/README.md) with PhpStorm
+installation.
+
+## How To Use It
+
+This image is ready for direct use within Coder Enterprise. Environments created
```
Comment (vapurrmaid) [inline images/phpstorm/README.md:12]:
```suggestion
This image is ready for direct use within Coder Enterprise. Workspaces created
```

File: images/phpstorm/Dockerfile.ubuntu [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+# Run everything as root
+USER root
+
+# Install php interpreter and CGI
+
+RUN apt-get update && \
+    DEBIAN_FRONTEND="noninteractive" apt-get install --yes \
+      php \
+      php7.4-cgi \
```
Comment (vapurrmaid) [inline images/phpstorm/Dockerfile.ubuntu:11]: **thought**: It might good for us to create a php base to extend from, since this version is hard-coded here

**Review by vapurrmaid — COMMENTED** [review]
This seems OK to me, going to let @jawnsy take a look as well.

File: images/phpstorm/README.md (hunk 1 above)
Comment (jawnsy) [reply images/phpstorm/README.md:12]: all of these images need some TLC, there's typos like that everywhere - I don't think we use the name "Coder Enterprise" anymore, either

Comment (vapurrmaid) [reply images/phpstorm/README.md:12]: Agreed. Should we just be consistent for now and ditch this suggestion?

**Review by jawnsy — APPROVED** [review]
seems reasonable for now... I think all of these images should be replaced though

**Comment by ericpaulsen** [comment]
closing this out, as JetBrains Projector is no longer supported.

**Pull request closed without merging**